pubspec.lock

```diff
@@ -497,13 +497,14 @@ packages:
     source: hosted
     version: "6.1.4"
   file_picker:
-    dependency: "direct main"
+    dependency: "direct overridden"
     description:
-      name: file_picker
-      sha256: "9d6e95ec73abbd31ec54d0e0df8a961017e165aba1395e462e5b31ea0c165daf"
-      url: "https://pub.dev"
-    source: hosted
-    version: "5.3.1"
+      path: "."
+      ref: master
+      resolved-ref: "6e9bd03c6884865f6958bdb86199c7dfaf459963"
+      url: "https://github.com/tddang-linagora/flutter_file_picker"
```

Comment:
If as a comment you can put the link to the upstream PR then that is even better!